com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/SampleTexture2DArrayNode.cs

```diff
@@ -37,6 +39,23 @@ public SampleTexture2DArrayNode()
             UpdateNodeAfterDeserialization();
         }
 
+        [SerializeField]
+        private bool m_DisableGlobalMipBias = false;
+
+        [ToggleControl("Disable Global Mip Bias")]
+        public ToggleData DisableGlobalMipBias
```

Comment:
This should probably be moved to the node inspector, rather than a first class control on the node, as it's a rather advanced control.  Would like to keep those a little more hidden to avoid confusing new users.

Reply:
That sounds good, I will look into it.